wikimedia/wikipedia-ios
# Test all AAALD navigations as pushes alternative

Fixes Phabricator tickets: 
[https://phabricator.wikimedia.org/T265674](https://phabricator.wikimedia.org/T265674)
[https://phabricator.wikimedia.org/T266833](https://phabricator.wikimedia.org/T266833)

**Test Steps**
- View changes, View discussion, Username, and Small event taps in the modal should now push on from the right rather than dismissing modal > push on from right on top of article.
- View changes and small event taps should go directly to the diff view, pushing on from the right. If small events are > 1, we show the diff as the compare (orange) view across all of those revisions, not the single revision view.
- For any snippet links that previously pushed on an article view, instead show the same article preview window as we do in the Edit Preview screen.
